libcxx/include/__utility/monostate.h

```diff
@@ -0,0 +1,66 @@
+// -*- C++ -*-
```

Comment:
I think we can and should just include existing `<__variant/monostate.h>` in `<utility>`.

Reply:
Yes, that makes sense